cmd/generate-config/config/config-openapi-spec.json

```diff
@@ -267,6 +268,23 @@
           "type": "string"
         }
       }
+    },
+    "storage": {
+      "description": "Storage represents a subfield of the microshift config data structure. Its purpose to provide a user\nfacing interface to control whether MicroShift should deploy LVMS on startup.",
+      "type": "object",
+      "properties": {
+        "driver": {
+          "description": "DriverOpt is an enum value that determines whether MicroShift deploys LVMS.",
+          "type": "string"
+        },
+        "with-csi-components": {
+          "type": "array",
+          "items": {
+            "description": "ComponentOpt values determine which CSI components MicroShift should deploy. Currently only csi snapshot components\nare supported.",
+            "type": "string"
```

Comment:
Should we specify allowed values here?

Reply:
ditto https://github.com/openshift/microshift/pull/3678/files#r1706414193

Reply:
Used `+kubebuilder:validation:items:Enum=[...]` to enforce values